src/renderer/components/Menu.tsx

```diff
@@ -68,10 +68,7 @@ export default function DeltaMenu(props: { selectedChat: FullChat }) {
     })
   const onVideoChat = () => {
     screenContext.openDialog('ConfirmationDialog', {
-      header: tx(
-        'videochat_invite_user_to_videochat',
-        selectedChat.contacts[0]?.displayName
-      ),
+      header: tx('videochat_invite_user_to_videochat', selectedChat.name),
```

Comment:
maybe another translation string is needed... 
"Invite %1$s to a video chat?" might work for now, but as the key ('videochat_invite_user_to_videochat') says its inviting a user I don't know if this still works in all translations, what do you think @r10s?

Reply:
actually, android uses chat.getName() already, so there is no issue in using the chat name, it shows:

"Invite MyGroup to a videochat?" which is fine